setup.py

```diff
@@ -31,6 +31,7 @@
 setup(
     name='emoji',
     author=author,
+    python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*',
     author_email=email,
     classifiers=[
         'Development Status :: 4 - Beta',
```

Comment:
Looking at this, I think we should also replace the development status classifier with `Development Status :: 5 - Production/Stable`